service/models.py

```diff
@@ -11,28 +11,36 @@
 # Create the SQLAlchemy object to be initialized later in init_db()
 db = SQLAlchemy()
 
-class DataValidationError(Exception):
+class Products(Exception):
```

Comment:
We cant have two classes with the name "Products" I think we can leave this as DataValidationError